### DISCUSSION TITLE
Plane cut?

### DISCUSSION THREAD

**Discussion by dwightkelly** [pmp-library/pmp-library#160] [General]
Does pmp have the ability to slice a mesh along a plane and fill the resulting hole like Meshmixer's Plane Cut feature?

**Comment by dsieger** [comment]
I'm afraid we don't. We have hole filling, but no slicing.

**Comment by dwightkelly** [comment]
There's the project - https://github.com/Nathaniell1/STL_CUT

**Reply by jellychen** [reply]
I think maybe I can implement this function

**Reply by FreakTheMighty** [reply]
@jellychen we need this function as well!
